UnicornRender/video/source/vulkan/Renderer.cpp

```diff
@@ -1309,6 +1304,12 @@ bool Renderer::CreateCommandBuffers()
 
             for(auto pVkMesh : m_vkMeshes)
             {
+                if (!pVkMesh->isVisible())
+                {
+                    ++j;
```

Comment:
Could you briefly explain why `j` shall be incremented here? It's used in `dynamicOffset` afterwards but if a mesh is invisible, is it still stored somewhere in a pipeline or how does this work?

Reply:
++j said that we just skip this mesh and move to next one. We just not rendering it.